documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
 In that sense, TeX will remain ahead of SILE for some time to come.
 But in terms of \em{core capabilities}, SILE aims at being at least equivalent to TeX.
 
+\section{SILE versus Typst and other new alternatives}
+
+Almost exactly one decade after SILE development started, a new competitor to LaTeX entered the scene with a bang:
+\footnote{Days of front-page coverage on Hacker News, Lobsters, and the like propelled it past 10k GitHub stars in the first 6 days of being public, a milestone SILE is still very far from reaching.}
+Typst.
+There is no shame in stating that Typst is a great piece of software and a worthy competitor to SILE as well as well as LaTeX.
+Typst is (very) fast, well documented, and relatively versatile.
+While both typesetting engines cover much of the same ground, the respective focus and implementation choices of each are so varied that direct comparisons don’t always have clear winners.
+At the end of the day the specific use case and workflow preferences are at least as relevant as the overall merits of each engine.
+
+When Typst was first conceived, SILE had been around the block in production for a while; but had only really been used for prose.
+Its math support was barely at a proof-of-concept stage and definitely not robust enough for scientific publishing work.
+This niche was exactly the felt need of Typst’s developers; the major focus right out of the gate was math and science papers.
+In the first releases it was more than a little bit rough in the prose department.
+Since then SILE’s math chops have greatly expanded to cover the majority of MathML.
+Typst’s feature set for handling prose have also improved.
+That being said the initial focus and energy of each project is still very visible.
+
+\begin{itemize}
+
+\item{\strong{Field of focus:}
+	SILE was initially conceived to handle hard problems in automated typesetting of books in many world orthographies.
+	Typst has some multi-lingual support but does not have robust support for the wide variety of typesetting issues posed by the world’s scripts and languages to the extent SILE does.
+	Typst was initially conceived for work in the academic sciences, and the robustness of the math and figure APIs directly reflects this.
+	SILE has math support but it has not been exercised to nearly the extent of Typst’s.}
+
+\item{\strong{Input formats:}
+	SILE was conceived to handle multiple input formats; out of the box it supports a LaTeX-like syntax but also includes an XML reader suitable for inputting other formats.
```

Comment:
Generic support for XML was never fully demonstrated in SILE's core distribution. I know: I suffered a lot on it. So this is a mere claim here, or a proven field with robust uses?

Reply:
On a second thought here:
- Typst has ways to process XML (https://typst.app/docs/reference/data-loading/xml/ - as well as several other structured formats (https://typst.app/docs/reference/data-loading/). 
- SILE is bundled with an XML parser, and the Lua ecosystem has libraries for some of these formats.

So they can both achieve the same kind of things - one with a custom API, the other by leveraging existing Lua components. I am not sure how it compares in term of input capabilities.